tools/robotick-launcher/tests/launcher/test_launcher_generate.py

```
@@ -83,12 +90,32 @@ def _list_files(dir_path: Path):
     return file_paths_rel
 
 
+def _normalized_fixture_files(dir_path: Path):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_normalized_fixture_files`

(ANN202)